spec/formula_checker.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.shared_context "formula checker", a => b do
+  let(:model) { described_class.new df, a, :logistic }
```

Comment:
@zverok How do I make `include_context` work? I do not know how to accept arugments and couldn't find easily it the internet. 

I'm using it via `include_context "formula checker", 'y~a:e' => %w[a e_B e_C].sort` but its saying `a` is not defined in `formula_checker.rb`.